R/augsynth.R

```diff
@@ -103,12 +99,34 @@ fit_augsynth_internal <- function(wide, synth_data, Z, progfunc,
     }
     ## fit augsynth
     if(progfunc == "Ridge") {
+        # 1. find hyperparameter using CV call, if lambda is null
+        # 2. call fit_ridgeuag_formatted with best CV (if-else case below, depending on scm=T or F)
+        # TODO: remove if else statement here since scm is an arg
         if(scm) {
```

Comment:
I don't think we need this if else block for `scm = TRUE` or `scm  = FALSE`, since it's a parameter below. Also right now it looks like if `scm = FALSE` and `lambda = NULL` the code will fail?

Reply:
Fixed, but fwiw when `scm=F` and `lambda=NULL`, the code defaults to the original ridge w/ CV (which has not been removed yet).